packages/fxa-settings/src/components/DataCollection/index.test.tsx

```diff
@@ -23,5 +30,19 @@ describe('DataCollection', () => {
     );
   });
 
-  // TODO: submission tests with mutations, FXA-4106
+  it('toggles', async () => {
+    const x = renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <DataCollection />
+      </AppContext.Provider>
+    );
+
+    const button = x.getByTestId('metrics-opt-out');
+    await button.click();
```

Comment:
You should be able to take out `await` from L41 and L45.

I think this could use a test for erroring but we can punt that to FXA-4106.